src/timer/timer_enable.cc

```diff
@@ -313,6 +317,9 @@ void TimerReport::PrintToLog(unsigned indent_level) const {
     using std::chrono::duration_cast;
     using std::chrono::nanoseconds;
```

Comment:
literal recommended.

Reply:
How? Can we do something like `GetXXXNumber()ns`?

Reply:
nvm